websites/F/feds/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "1105810827224809532",
+		"name": "for"
+	},
+	"service": "feds",
+	"description": {
+		"en": "feds.lol is a freemium biolink service."
+	},
+	"url": "feds.lol",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/TzIxLtq.png",
```

Comment:
This doesn't look like the official logo? (https://feds.lol/v4/img/logo.png)

Reply:
Sorry for the confusion. I am staff for feds.lol and can confirm that we are undergoing a rebrand that involves a changing of the logo. If you need me to prove this in some way I can.